charts/kyverno/values.yaml

```diff
@@ -71,6 +71,14 @@ resources:
     cpu: 100m
     memory: 50Mi
 
+initResources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
```

Comment:
Can we set the default memory limit to 256Mi? We saw there were cases the init container cleaned up stale RCRs, it's better to keep it aligned with the main container.